app/scss/main.scss

```diff
@@ -12,4 +12,6 @@
 @import 'loadout-builder';
 @import 'vendors';
 @import 'debug';
-@import './shell/materials-exchange'
\ No newline at end of file
+@import './shell/materials-exchange';
+@import './../vendor/components-font-awesome/css/font-awesome.css';
+@import './../vendor/angular-hotkeys/build/hotkeys.css'
```

Comment:
We can probably just import these from `node_modules` (they are dependencies expressed in the package.json, right?) using `@import '~angular-hotkeys/build/hotkeys.css'` or whatever the correct paths are. 

See [sass-loader imports](https://github.com/jtangelder/sass-loader#imports) for more.